documentation/src/main/asciidoc/chapter-4-retrieving-a-mapper.asciidoc

```diff
@@ -117,7 +117,7 @@ public interface CarMapper {
 ----
 ====
 
-The generated mapper will inject all classes defined in the **uses** attribute.
+The generated mapper will inject all classes defined in the **uses** attribute and MapStruct has detected that they are actually used on some of the mappings.
```

Comment:
does the injection also take place if a static method is used from the defined class?

Perhaps a better wording would be:
`The generated mapper will inject classes defined in the **uses** attribute if MapStruct has detected that it needs to use an instance of it for a mapping.`